src/ui/components/PocketItem.tsx

```diff
@@ -137,26 +163,35 @@ export const PocketItem = ({
 
   return (
     <div className={styles.item}>
-      <span className={styles.itemTitle}>
-        <PocketItemNoteLink
-          title={title}
-          noteExists={itemNoteExists}
-          onClick={async (event) => {
-            const clickAction = getPocketItemClickAction(event);
-            switch (clickAction) {
-              case PocketItemClickAction.NavigateToPocketURL:
-                navigateToPocketURL();
-                break;
-              case PocketItemClickAction.CreateOrOpenItemNote:
-                await createOrOpenItemNote(item);
-                break;
-              case PocketItemClickAction.Noop:
-                break;
-              default:
-                throw new Error(`Unknown PocketItemClickAction ${clickAction}`);
-            }
-          }}
+      <span className={styles.header}>
+        <span className={styles.itemTitle}>
+          <PocketItemNoteLink
+            title={title}
+            noteExists={itemNoteExists}
+            onClick={async (event) => {
+              const clickAction = getPocketItemClickAction(event);
+              switch (clickAction) {
+                case PocketItemClickAction.NavigateToPocketURL:
+                  navigateToItemURL();
+                  break;
+                case PocketItemClickAction.CreateOrOpenItemNote:
+                  await createOrOpenItemNote(item);
+                  break;
+                case PocketItemClickAction.Noop:
+                  break;
+                default:
+                  throw new Error(
+                    `Unknown PocketItemClickAction ${clickAction}`
+                  );
+              }
+            }}
+          />
+        </span>
+        <PocketItemExternalLink
+          title="Open in Pocket"
+          url={getPocketItemPocketURL(item)}
         />
+        <PocketItemExternalLink title="Open Original" url={item.resolved_url} />
```

Comment:
I think it might be clearer to the user to do the following:
1. Switch the order of these two links, so that the "Open Original comes first".
2. Rename "Open original" to simply "Open".

So it ends up looking like "Open" and "Open in Pocket", where "Open" is understood to be the existing behavior of opening the saved URL, and "Open in Pocket" is simply a modifier on top of that behavior that opens in Pocket.

LMK what you think.. It seems to me that this would be the easiest to understand for the user.

Reply:
Sure, I can make those changes.